src/source/Signaling/LwsApiCalls.c

```diff
@@ -643,6 +643,10 @@ STATUS lwsCompleteSync(PLwsCallInfo pCallInfo)
     return retStatus;
 }
 
+BOOL isLwsCallResultFailureRetryable(PCallInfo pCallInfo) {
+    return (STRSTR(pCallInfo->responseData, "Signature expired") == NULL);
```

Comment:
It is in fact guaranteed to be null terminated since the last commit on this branch:

https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/release_1_5_1/src/source/Signaling/LwsApiCalls.c#L119

But I made the change anyway just to be safe in case that changes in the future.